diff --git a/src/src/http_parser.cpp b/src/src/http_parser.cpp
index 1a6d9fa..32ac400 100644
--- a/src/http_parser.cpp
+++ b/src/http_parser.cpp
@@ -1536,14 +1536,6 @@ void parser::finish(http::message& http_msg) const
                                   http_request.get_content(),
                                   http_request.get_content_length()))
                 PION_LOG_WARN(m_logger, "Request form data parsing failed (POST urlencoded)");
-        } else if (content_type_header.compare(0, http::types::CONTENT_TYPE_MULTIPART_FORM_DATA.length(),
-                                               http::types::CONTENT_TYPE_MULTIPART_FORM_DATA) == 0)
-        {
-            if (! parse_multipart_form_data(http_request.get_queries(),
-                                            content_type_header,
-                                            http_request.get_content(),
-                                            http_request.get_content_length()))
-                PION_LOG_WARN(m_logger, "Request form data parsing failed (POST multipart)");
         }
     }
 }
